CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## Unreleased
+
+`Date`
+
+### What's new
+
+- **Added new `Breadcrumbs` component.**
+  - Supports customizing elements in `BreadcrumbItem` and handles overflow based on available space.
```

Comment:
Is this single subitem really needed?

Reply:
What?

Reply:
![image](https://user-images.githubusercontent.com/36186912/127163513-11cda474-44b0-4ff9-95c7-a622b92e6307.png)

Reply:
Oh, are you talking about the formatting? I initially had in the same line but it was too long.

Maybe I can break that down into two sub-items

Reply:
Just we are not using sub-items anywhere (maybe we should).

Reply:
https://github.com/iTwin/iTwinUI-react/blob/333640cec2f78702d3cf210aeaf9f60c238c2005/CHANGELOG.md#L80-L81

https://github.com/iTwin/iTwinUI-react/blob/333640cec2f78702d3cf210aeaf9f60c238c2005/CHANGELOG.md#L136-L138